db/seeds.d/02-roles_list.rb

```diff
@@ -53,5 +53,19 @@ def manage_organizations_permissions
     def view_permissions
       PermissionsList.permissions.select { |resource, name| name.start_with?('view_') }.map { |p| p.last.to_sym }
     end
+
+    def roles_descriptions
```

Comment:
Would it be possible to get the descriptions into `seeded_roles` method to have everything at one place?

Reply:
That would change the `RolesList.seeded_roles` format which might have been used by some plugin, but if you're fine with the change, I can change it and change both places where we use it in Foreman.

Reply:
Are you aware of any such a plugin? Unless there is serious reason for the backward compatibility, I vote for the change

Reply:
grepping all my local plugins didn't find anything, changing this and `SeedHelper#create_role` API